contrib/hook.sh

```diff
@@ -5,8 +5,9 @@ echo "WASP SWAP hook"
 set -x
 
 CG_PATH=$(jq -er '.linux.cgroupsPath' < config.json)
+POD_NAME=$(jq -er '.annotations["io.kubernetes.pod.name"]' < config.json)
 
-if [[ "$CG_PATH" =~ .*"burst".* ]];
+if [[ "$CG_PATH" =~ .*"burst".* && "$POD_NAME" =~ "virt-launcher".* ]];
```

Comment:
Workaround, because
```
-        "always": true
+        "annotations": {
+          "^io\\.kubernetes\\.pod\\.name$": "^virt-launcher.*"
```
in `hook.json` did not work.